tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,312 @@
+
+## Prerequisites
+
+- Log in to TiDB Cloud via [Password Authentication](/tidb-cloud/tidb-cloud-password-authentication.md) or [SSO Authentication](/tidb-cloud/tidb-cloud-sso-authentication.md), and then [Create a TiDB Cloud Dedicated Tier cluster](/tidb-cloud/create-tidb-cluster.md).
+
+- Set a password to access your cluster in secure settings.
+
+    To do so, you can navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, click **...** in the row of your Dedicated Tier cluster, and then select **Security Settings**. In security settings, you can click **Generate** to automatically generate a root password with a length of 16 characters, including numbers, uppercase and lowercase characters, and special characters.
+
+## Secure connection to a Dedicated Tier cluster
+
+In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of different connection methods and connect to your Dedicated Tier cluster as follows:
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, and then click the name of your Dedicated Tier cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner.  A dialog is displayed.
+
+3. On the **Standard Connection** tab of this dialog, follow the three steps to set up the TLS connctions.
+   - Step 1：Create traffic filter
+   - Step 2：Download TiDB cluster CA
+   - Step 3：Connect with an SQL client
+
+4. Under **Step 1：Create traffic filter** in the dialog, configure the IP addresses that are allowed to access your cluster. For more information, see [Configure an IP access list in standard connection](/tidb-cloud/configure-ip-access-list.md#configure-an-ip-access-list-in-standard-connection).
+
+5. Under **Step 2: Download TiDB cluster CA**, click **Download TiDB Cluster CA** to download it locally for client TLS configuration. Verifying the TiDB cluster CA can ensure the TLS connection is secure and reliable.
+
+6. Under **Step 3: Connect with an SQL client** in the dialog, click the tab of your preferred connection method, and then refer to the connection string and sample code on the tab to connect to your cluster.
+
+The following examples show the connection strings in MySQL, MyCLI , JDBC, Python, Go, and Node.js:
+
+<SimpleTab>
+<div label="MySQL CLI">
+
+MySQL CLI client attempts to establish a TLS connection by default. When you connect to Dedicated Tier clusters, you should set `ssl-mode` and `ssl-ca`.
+
+```shell
+mysql --connect-timeout 15 --ssl-mode=VERIFY_IDENTITY --ssl-ca=ca.pem --tls-version="TLSv1.2" -u root -h tidb.eqlfbdgthh8.clusters.staging.tidb-cloud.com -P 4000 -D test -p
+```
+
+Parameter description：
+
+- With `--ssl-mode=VERIFY_IDENTITY`, MySQL CLI client forces to enable TLS and validate TiDB Dedicated Tier clusters.
+- Use `--ssl-ca=<CA_root_path>` to set the CA root path on your system.
+- Use `--tls-version=TLSv1.2` to restrict the versions of TLS protocol.
```

Comment:
Do we need to mention that users can update 1.2 to 1.3 if they use TLS 1.3?

Reply:
good idea ， update now